rllib/offline/estimators/tests/test_ope.py

```diff
@@ -1,186 +1,204 @@
                 evaluation_config={"input": eval_data},
                 off_policy_estimation_methods={
                     "is": {"type": ImportanceSampling},
                     "wis": {"type": WeightedImportanceSampling},
-                    "dm_fqe": {
-                        "type": DirectMethod,
-                        "q_model_config": {"type": FQETorchModel},
-                    },
-                    "dr_fqe": {
-                        "type": DoublyRobust,
-                        "q_model_config": {"type": FQETorchModel},
-                    },
+                    "dm_fqe": {"type": DirectMethod},
+                    "dr_fqe": {"type": DoublyRobust},
                 },
             )
         )
         cls.algo = config.build()
 
-        # Train DQN for evaluation policy
-        for _ in range(n_episodes):
-            cls.algo.train()
-
         # Read n_episodes of data, assuming that one line is one episode
-        reader = JsonReader(eval_data)
-        cls.batch = reader.next()
-        for _ in range(n_episodes - 1):
-            cls.batch = concat_samples([cls.batch, reader.next()])
+        reader = DatasetReader(read_json(eval_data))
+        batches = [reader.next() for _ in range(n_episodes)]
+        cls.batch = concat_samples(batches)
         cls.n_episodes = len(cls.batch.split_by_episode())
         print("Episodes:", cls.n_episodes, "Steps:", cls.batch.count)
 
-        cls.mean_ret = {}
-        cls.std_ret = {}
-        cls.losses = {}
-
-        # Simulate Monte-Carlo rollouts
-        mc_ret = []
-        env = gym.make(env_name)
-        for _ in range(n_episodes):
-            obs = env.reset()
-            done = False
-            rewards = []
-            while not done:
-                act = cls.algo.compute_single_action(obs)
-                obs, reward, done, _ = env.step(act)
-                rewards.append(reward)
-            ret = 0
-            for r in reversed(rewards):
-                ret = r + cls.gamma * ret
-            mc_ret.append(ret)
-
-        cls.mean_ret["simulation"] = np.mean(mc_ret)
-        cls.std_ret["simulation"] = np.std(mc_ret)
-
     @classmethod
     def tearDownClass(cls):
-        print("Standalone OPE results")
-        print("Mean:")
-        print(*list(cls.mean_ret.items()), sep="\n")
-        print("Stddev:")
-        print(*list(cls.std_ret.items()), sep="\n")
-        print("Losses:")
-        print(*list(cls.losses.items()), sep="\n")
         ray.shutdown()
 
-    def test_is(self):
-        name = "is"
+    def test_ope_standalone(self):
+        # Test all OPE methods standalone
         estimator = ImportanceSampling(
             policy=self.algo.get_policy(),
             gamma=self.gamma,
         )
         estimates = estimator.estimate(self.batch)
-        self.mean_ret[name] = estimates["v_target"]
-        self.std_ret[name] = estimates["v_target_std"]
+        assert estimates is not None, "IS estimator did not compute estimates"
```

Comment:
You need to check for certain fields in the estimate dict right? Can you write a test private method to check the estimates dict if the logic can be shared between ope methods?